chart/env/prod.yaml

```diff
@@ -331,6 +331,11 @@ admin:
   replicas: 2
   service:
     type: NodePort
+  ingress:
+    enabled: true
+    annotations:
```

Comment:
do we need the same annotations in the staging.yaml env?

Reply:
if so, we could delete the reverse-proxy from Helm altogether

Reply:
Yes done.
I kept the reverse proxy for now in case we need to re-enable it. But once deployed, yes we should be able to remove it.